lib/light.py

```diff
@@ -0,0 +1,39 @@
+import RPi.GPIO as GPIO 
+
+class Light():
+            
+      def __init__(self, redlight = 18, bluelight = 11, greenlight = 16):
+            self.redlight = redlight
+            self.bluelight = bluelight
+            self.greenlight = greenlight
+            GPIO.setmode(GPIO.BCM)
+            GPIO.setwarnings(False)
+            GPIO.setup(redlight, GPIO.OUT)
+            GPIO.setup(bluelight, GPIO.OUT)
+            GPIO.setup(greenlight, GPIO.OUT)
+
```

Comment:
Do we need extra blank line here?

Reply:
The conventions for the linter we're using recommend 2 blank lines between functions

Reply:
2 blank lines between functions seems to be the convention for Python, according to the [style guide](https://www.python.org/dev/peps/pep-0008/#blank-lines)